### PR TITLE
Initial PR for Amazon Chime Notification component

### PR DESCRIPTION
**Pull request by danlindow**
This is part of a PR:
home-assistant/home-assistant#21603

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** https://github.com/home-assistant/home-assistant/pull/21603

## Checklist:

- [X ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/notify.amazon_chime [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    platform: amazon_chime
+    url: https://hooks.chime.aws/incomingwebhooks/some-room-id?token=some-secret-token
+
+```
+
+{% configuration %}
+url:
+  description: The full Chime Webhook URL. This includes "https" and the room ID and the token.
+  required: Required
+  type: string/url
```
Comment (klaasnicolaas) [inline source/_components/notify.amazon_chime]:
```suggestion
  type: string
```

File: source/_components/notify.amazon_chime [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+## {% linkable_title Configuration %}
+
+To use this notification platform in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+notify:
+  - name: Amazon Chime Teset
+    platform: amazon_chime
+    url: https://hooks.chime.aws/incomingwebhooks/some-room-id?token=some-secret-token
```
Comment (klaasnicolaas) [inline source/_components/notify.amazon_chime]:
```suggestion
    url: WEBHOOK_URL
```

File: source/_components/notify.amazon_chime [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+The URL that is generated for the webhook will need to be used inside the amazon_chime notification component.
+Further reading is available on [Amazon Chime documentation](https://docs.aws.amazon.com/chime/latest/ug/webhooks.html) pages.
+## {% linkable_title Configuration %}
+
+To use this notification platform in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+notify:
+  - name: Amazon Chime Teset
```
Comment (klaasnicolaas) [inline source/_components/notify.amazon_chime]:
```suggestion
  - platform: amazon_chime
```
You can remove the `name` variable because it is optional.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/notify.amazon_chime [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+Further reading is available on [Amazon Chime documentation](https://docs.aws.amazon.com/chime/latest/ug/webhooks.html) pages.
+## {% linkable_title Configuration %}
+
+To use this notification platform in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+notify:
+  - platform: amazon_chime
+    platform: amazon_chime
```
Comment (klaasnicolaas) [inline source/_components/notify.amazon_chime]: Remove this line

File: source/_components/notify.amazon_chime [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+sharing: true
+footer: true
+ha_category: Notifications
+ha_release: "TBD"
+---
+
+The `amazon_chime` notification platform enables publishing to an [AWS Chime](https://aws.amazon.com/chime/) webhook (chat room).
+
+## {% linkable_title Setup %}
+In order to utilize this component, a must first be signed into Amazon Chime.   Second, the user needs to create a Chime Webhook for the chatroom they are interested in having notifications from Home Assistant published.
```
Comment (klaasnicolaas) [inline source/_components/notify.amazon_chime]:
```suggestion
In order to utilize this component, a must first be signed into Amazon Chime. Second, the user needs to create a Chime Webhook for the chatroom they are interested in having notifications from Home Assistant published.
```

File: source/_components/notify.amazon_chime [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+{% configuration %}
+url:
+  description: The full Chime Webhook URL. This includes "https" and the room ID and the token.
+  required: Required
+  type: string
+{% endconfiguration %}
+
+### {% linkable_title Usage %}
+
+Amazon Chime is an online conferencing and communication tool by Amazon.  This service offers chat rooms for users to interact with.  This component leverages a webhook URL in order to send notifications from Home Assistant to a given chat room.
```
Comment (klaasnicolaas) [inline source/_components/notify.amazon_chime]:
```suggestion
Amazon Chime is an online conferencing and communication tool by Amazon. This service offers chat rooms for users to interact with. This component leverages a webhook URL in order to send notifications from Home Assistant to a given chat room.
```

File: source/_components/notify.amazon_chime [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  - platform: amazon_chime
+    platform: amazon_chime
+    url: WEBHOOK_URL
+
+```
+
+{% configuration %}
+url:
+  description: The full Chime Webhook URL. This includes "https" and the room ID and the token.
+  required: Required
```
Comment (klaasnicolaas) [inline source/_components/notify.amazon_chime]:
```suggestion
  required: true
```

File: source/_components/notify.amazon_chime [hunk 8]
```diff
@@ -0,0 +1,42 @@
+# ---
```
Comment (klaasnicolaas) [inline source/_components/notify.amazon_chime]:
```suggestion
---
```

File: source/_components/notify.amazon_chime [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+title: "Amazon Chime"
+description: "Instructions on how to send messages to Amazon Chime from Home Assistant."
+date: 2019-03-03 00:44
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Notifications
+ha_release: "TBD"
+---
```
Comment (klaasnicolaas) [inline source/_components/notify.amazon_chime:13]: Missing the `ha_iot_class`

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/notify.amazon_chime (hunk 9 above)
Comment (danlindow) [reply source/_components/notify.amazon_chime:13]: Is this required for a notification component?

**Comment by klaasnicolaas** [comment]
Parent PR still open

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed.

**Pull request closed without merging**